contracts/Scheduler/BaseScheduler.sol

```diff
@@ -20,9 +20,21 @@ contract BaseScheduler is SchedulerInterface {
     //------------------------
     // New API [WIP]
     //------------------------
+
+    /**
+     * @dev Schedules a new TransactionRequest using the 'simple' parameters.
+     * @param _toAddress The address destination of the transaction.
+     * @param _callData The bytecode that will be included with the transaction.
+     * @param _uintArgs [0] The callGas of the transaction.
+     * @param _uintArgs [1] The value of ether to be send with the transaction.
+     * @param _uintArgs [2] The size of the execution window of the transaction.
+     * @param _uintArgs [3] The (block or timestamp) of when the execution window starts.
+     * @param _uintArgs [4] The gasPrice which will be used to exeute this transaction.
+     * @return The address of the new TransactionRequest.
+     */
     function scheduleTxSimple(address _toAddress,
```

Comment:
I think this API should be removed in favor of a single scheduling API that *always* takes **all** scheduling parameters to require everything be set explicitly.